### PR TITLE
Make installation instruction more clear

### PR DESCRIPTION
**Pull request by baijum**

### CODE REVIEW

File: hack/install_devconsole/README.md [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
+    ./consoledeveloper.sh
+
+The script does the following things:
+
+1. Replaces the existing OpenShift console with the with a temporary
+   fork (talamer) based console
+2. Installs the DevConsole operator. You can see a message if it
+   already exists.
+3. Creates a non-admin user named `consoledeveloper` with the password
+   as `developer`. The user creates appropriate role bindings
```
Comment (dipak-pawar) [inline hack/install_devconsole/README.md]: @baiju it'll be good if we mention it clearly that `view` role is namespace scoped or cluster scoped.

Comment (Avni-Sharma) [reply hack/install_devconsole/README.md]: there are two roles self-provisioner and view, and both are cluster scoped. But does a non admin user really needs to know that?

File: hack/install_devconsole/README.md [hunk 2]
```diff
@@ -1,25 +1,39 @@
...
-The prerequisites for testing this are
-export KUBECONFIG=kubeconfig file
+    export KUBECONFIG=/path/to/kubeconfig
 
-Run the script consoledeveloper.sh
-It does the following:
-1. Replaces the existing openshift console with the talamer console
-2. Installs the operator. (Prompts if it already exists)
-3. Creates a non-admin user consoledeveloper with the password as developer with the suitable rolebinding(rolebinding being used here is self-provisioner and view)
+Also, ensure you have the latest `oc` command available in the `PATH`
```
Comment (rgarg1) [inline hack/install_devconsole/README.md]: Can we specify the minimum required version of _oc_ utility to be used?

File: hack/install_devconsole/README.md (hunk 1 above)
Comment (dipak-pawar) [reply hack/install_devconsole/README.md]: It's not about knowing anyone, it's about trying to be clear. If someone knows view role. first question come to his mind is that is it cluster scoped or namespace scoped. 

If we mention it clearly, in our documentation, I think user won't get confused

File: hack/install_devconsole/README.md [hunk 3]
```diff
@@ -1,25 +1,39 @@
...
-Logging in as the consoledeveloper user, you can now create a new project and do oc get csvs in the suitable namespace to see the installed operator.
-Expected Output-
-On the UI you can now see a consoledeveloper user under the kubeadmin option.
-You can enter the username as consoledeveloper and the password as developer here.
+    ./consoledeveloper.sh
+
+The script does the following things:
+
+1. Replaces the existing OpenShift console with the with a temporary
+   fork (talamer) based console
```
Comment (rgarg1) [inline hack/install_devconsole/README.md]: We can provide a link to the talamer repo for reference here.

File: hack/install_devconsole/README.md [hunk 4]
```diff
@@ -1,25 +1,39 @@
-##Non-admin user viewing the console with developer console perspective
+## Developer console for non-admin users
 
-Refers [https://jira.coreos.com/browse/ODC-347](https://jira.coreos.com/browse/ODC-347)
+You can use `consoledeveloper.sh` script available in this directory
+to enable developer console for a non-admin user.
```
Comment (rgarg1) [inline hack/install_devconsole/README.md]: We should also add to the list of pre-conditions that the script should be run to install developer perspective on an OCP4 cluster.

Comment (baijum) [reply hack/install_devconsole/README.md]: @Avni-Sharma Did we miss to specify some pre-condition?

Comment (Avni-Sharma) [reply hack/install_devconsole/README.md]: Cloning this repo locally, also as @rgarg1 suggested we can specify the min required version of oc

Comment (baijum) [reply hack/install_devconsole/README.md]: @Avni-Sharma I will make the changes and update the PR.

File: hack/install_devconsole/README.md (hunk 1 above)
Comment (baijum) [reply hack/install_devconsole/README.md]: Done. 34ec7dca8cb84c02acd0133139198be6d82cfe3c

File: hack/install_devconsole/README.md (hunk 2 above)
Comment (baijum) [reply hack/install_devconsole/README.md]: Done. 34ec7dca8cb84c02acd0133139198be6d82cfe3c

File: hack/install_devconsole/README.md (hunk 3 above)
Comment (baijum) [reply hack/install_devconsole/README.md]: Done. 34ec7dca8cb84c02acd0133139198be6d82cfe3c

File: hack/install_devconsole/README.md (hunk 4 above)
Comment (baijum) [reply hack/install_devconsole/README.md]: They see README and script after cloning the repo. So not required.

**Pull request merged**